gorm/README.md

```diff
@@ -27,3 +27,44 @@ db.password=dbpassword
 
 ```
 
+
+# Controller
+
+```
+type GormController struct {
+	revel.Controller
```

Comment:
Why are you embedding revel.Controller in this manner ? If this type was in the controllers folder you could embed the type directly inside another controller like
```
type MyController struct {
  gorm.GormController
}